test/clj_diff/test/core.clj

```diff
@@ -98,7 +98,13 @@
        "kitten" "sitting" 3
        "Saturday" "Sunday" 3
        "gumbo" "gambol" 2
-       "nBP8GaFHVls2dI8h9aK1FWdRgevf43" "925BCPcYhT5hs8L9T3K2T5C7U3Lz5v" 28
+       ;; TODO: The current levenshtein implementation incorrectly
```

Comment:
Discussed with @frenata. This issue turns out to be benign in terms of our interests, but I appreciate the comment to draw attention.